global.json

```
@@ -3,7 +3,7 @@
     "allowPrerelease": true
   },
   "tools": {
-    "dotnet": "6.0.200",
+    "dotnet": "6.0.408",
```

I think we should consider going down a feature band to minimize risk on this LTS servicing branch, but @marcpopMSFT may disagree.

```suggestion
    "dotnet": "6.0.311",
```

---

Sounds as good idea and should serve the purpose identically (same runtime) - so unless there will be no objections I'll flip it tomorrow

---

the .4xx band is fully in support, it's just not shipping in VS anymore. You can use either.

---

Yeah, my concern is more about using tools from "the future" relative to this branch, which is building what will go into 3xx if we need to ship. No specific reason to be concerned, I'm just conservative.

---

I updated to the lower band to be on a safe side